### ISSUE TITLE
[bug]: JSON Prettify adds trailing commas, causing invalid JSON

### ISSUE THREAD

**Issue by ninemind** [hoppscotch/hoppscotch#4411]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current behavior

When using the "Prettify" feature on JSON in the in the request body, trailing commas are added after the last property in an object, resulting in an invalid JSON format that does not conform to the JSON standard.

### Steps to reproduce

1. Open `Raw Request Body`
2. Set `Content-Type` to `application/json`
3. Enter a simple JSON object, for example:
```
{
  "test": 123
}
```
4. Click on the "Prettify" button to format the JSON.
5. Observe that the prettified output contains a trailing comma, such as:
```
{
  "test": 123,
}
```
Notice that the resulting JSON is not valid according to the JSON standard.

### Environment

Production

### Version

Self-hosted

**Comment by ninemind** [comment]
The editor does not provide any indication that the resulting JSON is invalid after adding trailing commas. This lack of validation can lead to confusion, as the JSON appears correct but causes issues when sent in a request. Specifically, many APIs reject the request body or fail to process it correctly due to the invalid trailing commas, leading to unexpected errors.

**Comment by yashs33244** [comment]
can i solve this issue ..?

**Comment by adelrodriguez** [comment]
Can confirm, just ran into this issue as well after updating to the latest version.

**Comment by Pranay-Pandey** [comment]
Can this be solved by removing the DEFAULT_OPTIONS for formatting in the file 

`packages/hoppscotch-common/src/helpers/editor/linting/jsoncPretty.ts`

```
const DEFAULT_OPTIONS: Required<PrettifyOptions> = {
  indent: 2,
  maxLength: 80,
  commentSpace: true,
  trailingComma: true,
}
```

Why do we have trailingComma as true, instead of false?

**Comment by NayamAmarshe** [comment]
Please fix this!

**Comment by subframe7536** [comment]
Please set the default `trailingComma` to `false` 🙏

**Comment by sscotth** [comment]
Prettier has a native JSON parser that handles JSON correctly. The selected content type (XML, HTML, JSON, etc...) should change the prettier parser to match.

**Comment by jamesgeorge007** [comment]
#4416 released in [v2024.10.0](https://github.com/hoppscotch/hoppscotch/releases/tag/2024.10.0).
